applications/connectivity_bridge/src/modules/power_handler.c

```diff
@@ -16,7 +16,9 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_BRIDGE_POWER_MGMT_LOG_LEVEL);
 
-static struct k_delayed_work power_down_work;
+static void power_down_handler(struct k_work *work);
+
+K_WORK_DELAYABLE_DEFINE(power_down_work, power_down_handler);
```

Comment:
Add `static` before this to prevent `power_down_work` from becoming a global symbol.

Reply:
Changed to static as suggested